proposal.txt

```diff
@@ -0,0 +1,7 @@
+i-- Oscar So: Proposal --
+I will be trying to create an unlimited calendar, where the user inputs which year and month, and the    n outputs the calendar for the chosen time period. If this fails, i will try to make a calendar for o    nly this year. i will use functions outside of main and call the functions throughout the project as     I think that calculating where the days are located (eg. monday, tuesday) for a given date will requi    re a lot more coding. I will also use arrays and loops to print out the calendar after scanning for t    he user input. i chose this project because I feel that this project is challenging because of the po    sitioning of the dates in the calendar given a random inputted month and year, eg aug 2158?.
+SUN MON TUE WED THU FRI SAT
+                        1
+2   3   4   5   6   7   8
+9   10  11  12  13  14  15
+........... etc.
```

Comment:
This seems a little bit limited for a project.  Perhaps you can add functionality?  For instance, you may include this as part of a planner program that allows users to enter events for the days they input after displaying the calendar for the month/year.

Please expand upon your proposal to encompass a slightly larger project.